app/helpers/arclight_helper.rb

```diff
@@ -113,6 +113,19 @@ def display_size(size)
     size.to_s
   end
 
+  # determine which icon to show in search results header
+  # these icon names will need to be updated when the icons are determined
+  def document_header_icon(document)
+    level = document.level || nil
```

Comment:
I don't think we need to do the `nil` check here anymore as the accessor in `SolrDocument` handles that.

What do you think about switching this to a case statement as opposed to the nested if/else?

Reply:
Refactored to case statement.